mesonbuild/options.py

```diff
@@ -265,6 +265,11 @@ def printable_value(self) -> T.Union[str, int, bool, T.List[T.Union[str, int, bo
         assert isinstance(self.value, (str, int, bool, list))
         return self.value
 
+    def printable_choices(self) -> T.Optional[T.List[str]]:
+        if not self.choices:
+            return None
+        return [str(c) for c in self.choices]
```

Comment:
It seems that `self.choices` could be a `str`. In that case, that would result in a list of letters, isn't it?

Reply:
Yeah, at the point it could. So I'll fix that.